stablehlo/dialect/StablehloOps.td

```diff
@@ -356,8 +356,8 @@ def StableHLO_FloorOp: StableHLO_UnaryElementwiseOp<"floor",
 
 def StableHLO_ImagOp: StableHLO_UnaryElementwiseOp<"imag",
     [Pure, DeclareOpInterfaceMethods<InferTypeOpInterface>],
-    HLO_FpOrComplexTensor, HLO_FpTensor> {
-  let summary = "Imag operator";
+    HLO_FpOrComplexTensor /*imag_i1*/, HLO_FpTensor> {/*imag_c1*/
+  let summary = "Imag operation";
```

Comment:
Add input constraints as per  the checklist.

Reply:
For unary ops, I have the comment `imag_c1` in line 502 since it is covered by the `StableHLO_UnaryElementwiseOp` class.